File: Estrutura de Dados/Par_ou_Impar.py
def par_ou_impar(numero):
    if numero % 2 ==0:
        return "PAR"
    else:
        return"IMPAR"

numero = float(input("Digite um número: "))

resultado = par_ou_impar(numero)

print(f"O número {numero} é {resultado}!")

File: Estrutura de Dados/Par_ou_Impar.c
#include <stdio.h>

Int ParOuImpar(int numero)
{
    if (numero % 2 == 0)
    {
        return 1;
    }
    else
    {
        return 0;
    }
    

}

int main()
{
    int numero;
    int resultado;

    printf(float("Digite um número: "))
    scanf("%d", &numero);    //& Diz ao scanf onde guardar o valor


    resultado = ParOuImpar(numero):
    if(resultado ==1)
    {
        print("PAR")
    }
    else
    {
        print("IMPAR")
    }
    return 0;
    
}
